lib/geordi/util.rb

```diff
@@ -200,7 +200,7 @@ def irb_version
       end
 
       def ruby_version
-        testing? ? Gem::Version.new(ENV['GEORDI_TESTING_RUBY_VERSION']) : RUBY_VERSION
+        Gem::Version.new(testing? ? ENV['GEORDI_TESTING_RUBY_VERSION'] : RUBY_VERSION)
```

Comment:
To have 1 line per action, please split this into two lines: deriving the version, and wrapping it.